### PR TITLE
✨ clinical submission schema download

### PR DESCRIPTION
**Pull request by hlminh2000**
**Description of changes**

Implements the tsv template downloads
Also creates a `DropdownButton` component. Preview of this is available at: [here](https://5daa1ee36b66560008ba05fd--argo-ui-storybook.netlify.com/?path=/story/uikit-dropdownbutton--basic) (FYI @kcullion)

**Type of Change**

- [ ] Bug
- [x] Styling
- [x] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [x] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

**Comment by kcullion** [comment]
Thanks @hlminh2000 

Can you just make the dropdown values font-weight:normal (not bold)
And make sure they are using the data typography style (font-size:12px) and maybe make the min-width of the dropdown a bit wider...I'm hoping "Primary Diagnosis" won't wrap to two lines

Screen Shot 2019-10-18 at 4 35 38 PM

**Comment by hlminh2000** [comment]
> Thanks @hlminh2000
> 
> Can you just make the dropdown values font-weight:normal (not bold)
> And make sure they are using the data typography style (font-size:12px) and maybe make the min-width of the dropdown a bit wider...I'm hoping "Primary Diagnosis" won't wrap to two lines
> 
> Screen Shot 2019-10-18 at 4 35 38 PM

It's not built into the component, only set to bold as a code demo of how to put custom stuff in there if needed :P

File: components/pages/submission-system/program-clinical-submission/Instruction/FileTemplatesDownloadButton.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  downloadTsvFileTemplate,
+} from '../../common';
+import { css } from 'uikit';
+import DropdownButton from 'uikit/DropdownButton';
+import { capitalize } from 'global/utils/stringUtils';
+
+export default ({ clinicalTypes }: { clinicalTypes: string[] }) => {
+  const onItemClick: React.ComponentProps<typeof DropdownButton>['onItemClick'] = item => {
+    if (item.value === 'all') {
+      downloadTsvFileTemplate(`all.zip`);
```
Comment (joneubank) [inline components/pages/submission-system/program-clinical-submission/Instruction/FileTemplatesDownloadButton.tsx]: I don't think we need the `.zip` anymore, Jaser updated clinical to provide proper file naming for the zip file, so it won't be dependant on the URL for the filename anymore

Comment (hlminh2000) [reply components/pages/submission-system/program-clinical-submission/Instruction/FileTemplatesDownloadButton.tsx]: cool! will update :)

**Pull request merged**